osdep/terminal-unix.c

```diff
@@ -235,7 +235,7 @@ static void process_input(struct input_ctx *input_ctx, bool timeout)
             unsigned char c = buf.b[0];
             skip_buf(&buf, 1);
             if (c < 32) {
-                c = c <= 25 ? (c + 'a' - 1) : (c - 25 + '2' - 1);
+                c = c <= 26 ? (c + 'a' - 1) : (c - 26 + '2' - 1);
```

Comment:
I think this introduces a bug where ^3-^7 are now one-off.

Would changing to this be OK (including the comment)?
```c
// 1..26 are ^A..^Z, 27..31 are ^3..^7
c = c <= 26 ? (c + 'a' - 1) : (c + '3' - 27);
```

And you can add at the commit message that `(c - 25 + '2' - 1)` was correct, but it was rephrased for clarity.

Reply:
Oops, yeah. I updated the commit.